examples/raycast.py

```diff
@@ -49,21 +49,32 @@ def random_pos(width, height):
 
 running = True
 
-ray_endpoint = pygame.Vector2()
 ray_count = 360
 
+directions = [(math.cos(angle), math.sin(angle)) for x in range(ray_count)]
```

Comment:
Don't you see the issue here? you are basically doing calculations in python that could've easily been done in C with just the angle parameter, Same for line 72.